### PR TITLE
Build scripts for maistra/envoy & envoyproxy/envoy repo

### PR DESCRIPTION
**Pull request by ManirajDeivendran**
Created build script for maistra/envoy & envoyproxy/envoy.

maistra/envoy build Logs: [maistra_envoy_ubi_8.4_build_logs.txt](https://github.com/ppc64le/build-scripts/files/7264374/maistra_envoy_ubi_8.4_build_logs.txt)
envoyproxy/envoy build Logs: [envoyproxy_envoy_ubi_8_4_build_logs.txt](https://github.com/ppc64le/build-scripts/files/7264378/envoyproxy_envoy_ubi_8_4_build_logs.txt)

### CODE REVIEW

File: e/envoyproxy/envoyproxy_envoy_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+cd $SOURCE_ROOT
+git clone https://gn.googlesource.com/gn
+cd gn
+git checkout 992e927e217baa8a74e6e2c5d7417cb65cf24824
+python build/gen.py
+ninja -C out
+ln -sf $SOURCE_ROOT/gn/out/gn /usr/local/bin/gn
+
+#Install Bazel binary
+curl -O https://oplab9.parqtec.unicamp.br/pub/ppc64el/bazel/ubuntu_18.04/bazel_bin_ppc64le_3.7.2
```
Comment (ghatwala) [inline e/envoyproxy/envoyproxy_envoy_ubi_8.4.sh:76]: @ManirajDeivendran  - can't we use bazel rpms here instead as we validate this on UBI.

File: m/maistra/maistra_envoy_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+cd $SOURCE_ROOT
+git clone https://gn.googlesource.com/gn
+cd gn
+git checkout 992e927e217baa8a74e6e2c5d7417cb65cf24824
+python build/gen.py
+ninja -C out
+ln -sf $SOURCE_ROOT/gn/out/gn /usr/local/bin/gn
+
+#Install Bazel binary
+curl -O https://oplab9.parqtec.unicamp.br/pub/ppc64el/bazel/ubuntu_18.04/bazel_bin_ppc64le_3.7.2
```
Comment (ghatwala) [inline m/maistra/maistra_envoy_ubi_8.4.sh:76]: bazel installation steps could help - https://openpower.ic.unicamp.br/post/installing-bazel-from-repository/

**Review by ghatwala — CHANGES_REQUESTED** [review]

File: e/envoyproxy/envoyproxy_envoy_ubi_8.4.sh (hunk 1 above)
Comment (ManirajDeivendran) [reply e/envoyproxy/envoyproxy_envoy_ubi_8.4.sh]: Resolved by using bazel rpms.

**Review by ghatwala — APPROVED** [review]

**Comment by ghatwala** [comment]
LGTM

**Pull request merged**